api/call_service.go

```diff
@@ -35,11 +41,12 @@ func (s *Server) accountBalances(ctx context.Context, params map[string]interfac
 		return nil, xerr
 	}
 	var (
-		err    error
-		hash   []byte
-		height uint64
```

Comment:
Why height is not needed any more?

Reply:
Because we're no longer making a separate Access API call to `GetAccountAtBlockHeight` within `s.getOnchainData` any more.

Instead, we get both normal account balances and proxy account balances by executing a single transaction script using the Access API `ExecuteScriptAtBlockID` call.